packages/api/controllers/nomination.js

```diff
@@ -66,7 +67,17 @@ const updateNomination = async (req, res) => {
     const nomination = await db.Nomination.update(
       { status: req.body.status },
       { where: { id }, returning: true }
-    );
+    ).catch ((err)=> {
+      console.log('Nomination Not Found', err)
+      return res.status(400)});
+
+    const updatedNom = nomination[1][0].dataValues
+    //updated nom is being captured under updatedNom, can continue using additional conditional to use other email functions,
+    //depending on status of application
+    //current nominations don't have decline status, that should come after nominations hit ready for board review. TBD
+    if (updatedNom.status === 'Decline') {
```

Comment:
I think there is an issue here that if a declined application is updated again they will recieve the email again. 

I think we can use [changed[(https://sequelize.org/v3/api/instance/) to make this better. 

Something like `if updatedNom.changed('status') && updatedNom.status == 'Declined' will be better